spec.html

```diff
@@ -38904,7 +38904,7 @@ <h1>reads-bytes-from</h1>
       <h1>reads-from</h1>
       <p>For a candidate execution _execution_, _execution_.[[ReadsFrom]] is the least Relation on events that satisfies the following.</p>
       <ul>
-        <li>For each pair (_R_, _W_) in SharedDataBlockEventSet(_execution_), (_R_, _W_) is in _execution_.[[ReadsFrom]] if _W_ is in _execution_.[[ReadsBytesFrom]](_R_).</li>
+        <li>For each pair (_R_, _W_) in SharedDataBlockEventSet(_execution_), (_R_, _W_) is in _execution_.[[ReadsFrom]] if and only if _W_ is in _execution_.[[ReadsBytesFrom]](_R_).</li>
```

Comment:
This bullet was already correct, because reads-from is defined as the "least relation".

In my opinion, either reads-from should be defined as an exact relation such that... "if and only if" (like agent-order), or the least relation such that... "if" (the old wording). Currently this mixes the two, so there is redundant language.

Reply:
I will make it an exact relation.